ci/ci

```diff
@@ -141,37 +233,92 @@ def commit_docker_image():
                 cmd = 'docker rmi ' + bad_images
                 run_cmd(cmd)
         except Exception as e:
-            log_error(e)
+            log_warning(
+                "could not remove bad (dangling) images, please remove manually")
+            print(e)
+    return True
 
 
-def get_pkgs(platform):
-    build_cmd = '/src/build-pbs-packages.sh'
-    if platform != 'called':
-        try:
-            subprocess.run([pbs_dirname + '/ci', '--delete'])
-            subprocess.run([pbs_dirname + '/ci', '--start=' + platform])
-        except:
-            log_error('Failed to restart ci.')
-    log_info('The package build logs can be found in logs/pkglogs')
-    run_docker_cmd(
-        'docker-compose exec pbs-build bash -c \' ' + build_cmd + ' | tee /logs/pkglogs \' ')
+def create_ts_tree_json():
+    benchpress_opt = os.path.join(
+        ci_dirname, '.config_dir', '.benchpress_opt')
```

Comment:
please use macros for `.config_dir`, '.benchpress_opt' and such hard coded strings in whole file